eth2util/eth2exp/attagg.go

```diff
@@ -154,27 +154,39 @@ func (b *BeaconCommitteeSubscription) String() (string, error) {
 
 // BeaconCommitteeSubscriptionResponse is the response from beacon node after submitting BeaconCommitteeSubscription.
 type BeaconCommitteeSubscriptionResponse struct {
-	// ValidatorIndex is the index of the validator that made the subscription request.
+	// ValidatorIdex is the index of the validator making the subscription request.
 	ValidatorIndex eth2p0.ValidatorIndex
+	// Slot is the slot for which the validator is attesting.
+	Slot eth2p0.Slot
+	// CommitteeIndex is the index of the committee of which the validator is a member at the given slot.
+	CommitteeIndex eth2p0.CommitteeIndex
+	// CommitteesAtSlot is the number of committees at the given slot.
+	CommitteesAtSlot uint64
 	// IsAggregator indicates whether the validator is an attestation aggregator.
 	IsAggregator bool
+	// SelectionProof is the slot signature proving the validator is an aggregator for this slot and committee.
+	SelectionProof eth2p0.BLSSignature
```

Comment:
we also need to update https://github.com/ethereum/beacon-APIs/pull/224/files with the updated response structure later

Reply:
yup